src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -709,14 +723,18 @@ export default class CanvasController {
      * @param aircraft
      */
     canvas_draw_separation_indicator(cc, aircraft) {
-        if (!GameController.shouldUseTrailingSeparator(aircraft)) {
+        if (!GameController.shouldUseTrailingSeparationIndicator(aircraft)) {
             return;
         }
 
         const runway = aircraft.fms.currentRunway;
         const oppositeOfRunwayHeading = runway.angle + Math.PI;
```

Comment:
is there a method on the `RunwayModel` that does this already?  if there isn't, there should be 😉